tests/test_eos.py

```diff
@@ -71,6 +97,41 @@ def test_reference_values(self):
             Grun_test = i.grueneisen_parameter(pressure, temperature, rock.params['V_0'], rock.params)
             self.assertFloatEqual(Grun_test, rock.params['grueneisen_0'])
 
+    def test_reference_values_noG(self):
```

Comment:
Why the different temperatures (300., 0.) in this test? The temperature shouldn't matter, surely.
If you have a reference pressure as 0. (rather than 1.e5) Pa, then this should be in the params as P_0, and you should test that the Vinet EoS handles different reference pressures in the same way that MT and BM do.

IMO, this should be made into at least two tests; one for Vinet, and one for BM4. 

Another sensible (and extremely useful) test for BM4 would be to set K'' to the value inferred by BM3, and make sure that the results are the same at some high pressure. Same goes for BM3 wrt. BM2, if we don't have that test already.